Assets/Tests/InputSystem/CoreTests_ProjectWideActions.cs

```diff
@@ -29,12 +81,11 @@ internal partial class CoreTests
     // Note that any existing default created asset is preserved during test run by moving it via ADB.
 
     const string TestCategory = "ProjectWideActions";
-    const string kAssetBackupDirectory = "Assets/~TestBackupFiles";
-    const string kDefaultProjectWideAssetBackupPath = kAssetBackupDirectory + "/DefaultProjectWideAssetBackup.json";
+    const string m_AssetBackupDirectory = "Assets/~TestBackupFiles";
+    const string s_DefaultProjectWideAssetBackupPath = "Assets/~TestBackupFilesDefaultProjectWideAssetBackup.json";
```

Comment:
Looks like a slash is missing before DefaultProjectWideAssetBackup

Reply:
Maybe it should be there instead of append on next line, will update

Reply:
Fixed in latest commit.